ghost/portal/src/components/Frame.styles.js

```diff
@@ -263,7 +263,7 @@ const FrameStyles = `
 }
 
 .gh-portal-popup-background.preview {
-    background: linear-gradient(45deg, rgba(255,255,255,1) 0%, rgba(249,249,250,1) 100%);
```

Comment:
This is used in Ghost Admin only, the colors here are not supposed to be changed. Let's keep it hardcoded